docs/404.html

```diff
@@ -99,7 +99,7 @@
   aria-label="Search this site…"
   autocomplete="off"
   
-  data-offline-search-index-json-src="/aws-sdk-go-v2/offline-search-index.30b245754e63db1f94dcbaa3ba0f9915.json"
+  data-offline-search-index-json-src="/aws-sdk-go-v2/offline-search-index.1ed5201c9429d6d49ab1e6d6c5e5409e.json"
```

Comment:
Were you able to confirm if this offline search works?

Reply:
I did actually check this and it worked for me both in the preview _and_ on the current render of the public site. We'll see what happens when this merges and gets propagated. The search index appears to get a new filename every time, it's possible that it just went out of sync in a previous pass on re-generating.

Reply:
cc @stefansundin